hibernate-core/src/main/java/org/hibernate/type/OneToOneType.java

```diff
@@ -55,19 +56,22 @@ public OneToOneType(
 			String uniqueKeyPropertyName,
 			boolean lazy,
 			boolean unwrapProxy,
+			boolean nullable,
```

Comment:
https://github.com/hibernate/hibernate-orm/pull/2451

Reply:
Yes please follow the same pattern used for the deprecated constructor above.